metron-deployment/roles/metron-builder/tasks/main.yml

```diff
@@ -15,4 +15,7 @@
 #  limitations under the License.
 #
 ---
-hdp_repo_def: "http://public-repo-1.hortonworks.com/HDP/centos6/2.x/updates/2.5.0.0/hdp.repo"
+- name: Build Deployment Artifacts
+  local_action: shell cd {{ metron_build_dir }} && mvn clean package -DskipTests -P HDP-2.5.0.0,mpack,build-rpms
```

Comment:
This is a "nice to have", but I noticed is that if you run "Full Dev" `vagrant up` without Docker running, it fails at this step with an awfully long and convoluted error message.  

```
...
ed(Launcher.java:289)", "\tat org.codehaus.plexus.classworlds.launcher.Launcher.launch(Launcher.java:229)", "\tat org.codehaus.plexus.classworlds.launcher.Launcher.mainWithExitCode(Launcher.java:415)", "\tat org.codehaus.plexus.classworlds.launcher.Launcher.main(Launcher.java:356)", "[INFO] ------------------------------------------------------------------------", "[INFO] Reactor Summary:", "[INFO] ", "[INFO] Metron ............................................. SUCCESS [  0.327 s]", "[INFO] metron-analytics ................................... SUCCESS [  0.002 s]", "[INFO] metron-maas-common ................................. SUCCESS [  5.415 s]", "[INFO] metron-platform .................................... SUCCESS [  0.001 s]", "[INFO] metron-test-utilities .............................. SUCCESS [  3.629 s]", "[INFO] metron-integration-test ............................ SUCCESS [  7.044 s]", "[INFO] metron-maas-service ................................ SUCCESS [  3.743 s]", "[INFO] metron-common ...................................... SUCCESS [ 18.669 s]", "[INFO] metron-statistics .................................. SUCCESS [ 13.954 s]", "[INFO] metron-hbase ....................................... SUCCESS [  2.471 s]", "[INFO] metron-profiler-common ............................. SUCCESS [  0.754 s]", "[INFO] metron-profiler-client ............................. SUCCESS [ 15.322 s]", "[INFO] metron-profiler .................................... SUCCESS [ 32.045 s]", "[INFO] metron-writer ...................................... SUCCESS [ 12.181 s]", "[INFO] metron-enrichment .................................. SUCCESS [ 39.843 s]", "[INFO] metron-indexing .................................... SUCCESS [ 12.021 s]", "[INFO] metron-solr ........................................ SUCCESS [ 14.621 s]", "[INFO] metron-pcap ........................................ SUCCESS [  1.248 s]", "[INFO] metron-parsers ..................................... SUCCESS [ 40.486 s]", "[INFO] metron-pcap-backend ................................ SUCCESS [ 22.573 s]", "[INFO] metron-data-management ............................. SUCCESS [ 45.768 s]", "[INFO] metron-api ......................................... SUCCESS [ 20.498 s]", "[INFO] metron-management .................................. SUCCESS [  4.632 s]", "[INFO] elasticsearch-shaded ............................... SUCCESS [  3.916 s]", "[INFO] metron-elasticsearch ............................... SUCCESS [ 44.956 s]", "[INFO] metron-deployment .................................. SUCCESS [  0.000 s]", "[INFO] metron-rpm ......................................... FAILURE [ 10.066 s]", "[INFO] Metron Ambari Management Pack ...................... SKIPPED", "[INFO] metron-docker ...................................... SKIPPED", "[INFO] ------------------------------------------------------------------------", "[INFO] BUILD FAILURE", "[INFO] ------------------------------------------------------------------------", "[INFO] Total time: 06:16 min", "[INFO] Finished at: 2017-03-06T19:09:39-05:00", "[INFO] Final Memory: 162M/1993M", "[INFO] ------------------------------------------------------------------------", "[ERROR] Failed to execute goal org.codehaus.mojo:exec-maven-plugin:1.5.0:exec (docker-build) on project metron-rpm: Command execution failed. Process exited with an error: 1 (Exit value: 1) -> [Help 1]", "[ERROR] ", "[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.", "[ERROR] Re-run Maven using the -X switch to enable full debug logging.", "[ERROR] ", "[ERROR] For more information about the errors and possible solutions, please read the following articles:", "[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/MojoExecutionException", "[ERROR] ", "[ERROR] After correcting the problems, you can resume the build with the command", "[ERROR]   mvn <goals> -rf :metron-rpm"], "warnings": []}
```

I think this might cause some headaches, especially for new folks.  Ideally, we could add some kind of more intelligible error message.